src/docs/export.md

```diff
@@ -177,6 +177,12 @@ The following is an abbreviated export file from a command in the pants repo:
 
 # Export Format Changes
 
+## 1.0.15
+
+Added 'runtime_platform' to target
+
+Deprecated 'test_platform'. (todo figure out what deprecation means)
```

Comment:
cc @wisechengyi

Reply:
1. Check the Intellij plugin repo and there is no usage on this field 
2. Sanity checked import a big projects in Source. Worked fine as well.

So I think we can just remove it in this PR. the risk is low. If any, the repair cost should be low as well.

Reply:
I'll remove it and bump the version to`1.1.0`